ToDoApp/ToDoAppTests/ToDoServiceTests.swift

```diff
@@ -56,4 +61,21 @@ final class ToDoServiceTests: XCTestCase {
         }
         XCTAssertEqual(failure, RepositoryError.createOrUpdateError)
     }
+    
+    func testGetTask_Success() async {
+        let repositoryMock: RepositoryMock<TaskModel> = RepositoryMock(createOrUpdate: { value in
+            // Mock repository behaviour returns nil error
+            return RepositoryError.createOrUpdateError
+        }, read: {
+            return .success([TaskModel(id: UUID(), title: "Hi", deadline: Date(timeIntervalSince1970: 1672558922))])
+        })
+        let todoService = ToDoServiceImpl(dateChecker: Calendar.current, repo: repositoryMock)
+        let result = await todoService.getTasks()
+        guard case .success(let todoModel) = result else {
+            XCTFail("Result expected to be success")
+            return
+        }
+        XCTAssertEqual(todoModel.other[0].title, "Hi")
```

Comment:
It would be better if todomodel would conform to equatable protocol and assertion would be done on the whole value
XCTAssertEqual (todoModel, .init(other: [model]))

In case if new properties would be added to the Section model test will verify it as well and we wouldn't need to remember that we have a test that needs to be updated as well